test/es-module/test-esm-initialization.mjs

```diff
@@ -8,22 +8,22 @@ import { describe, it } from 'node:test';
 describe('ESM: ensure initialization happens only once', { concurrency: true }, () => {
   it(async () => {
     const { code, stderr, stdout } = await spawnPromisified(execPath, [
+      '--experimental-import-meta-resolve',
```

Comment:
Why is this being added? I thought we removed this flag, or were going to do so immediately after landing off-thread.

Reply:
I think it was forgotten when the test was added, `runmain.js` use `import.meta.resolve`.

Reply:
Okay so this is just an unrelated change? Should this be in its own PR?